libPhoneNumber/NBPhoneNumberUtil.m

```diff
@@ -912,12 +908,12 @@ - (NSString *)normalizeHelper:(NSString *)sourceString
 
   for (NSUInteger i = 0; i < numberLength; ++i) {
     unichar character = [sourceString characterAtIndex:i];
-    NSString *newDigit = [normalizationReplacements
-        objectForKey:[[NSString stringWithFormat:@"%C", character] uppercaseString]];
+    NSString *charString = [NSString stringWithCharacters:&character length:1];
```

Comment:
How about replacing 910 and 911 with NSString *charString = [sourceString substringWithRange:NSMakeRange(i, 1)];

Reply:
Done